Analyzer/test/condor/condorSubmit.py

```diff
@@ -61,7 +61,7 @@ def main():
 
     if os.path.isdir(workingDir):
         print red("Job directory already exists and cannot proceed safely ! Exiting...")
-        exit(0)
+        #exit(0)
```

Comment:
Did you mean to push this? Normally we don't want to overwrite stuff.

Reply:
Indeed, seems like some intermixed changes